docs/apidiffs/current_vs_latest/opentelemetry-extension-kotlin.txt

```diff
@@ -1,2 +1,10 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
+===  UNCHANGED CLASS: PUBLIC FINAL io.opentelemetry.extension.kotlin.ContextExtensionsKt  (not serializable)
```

Comment:
Wonder if we need to disable jcmp for kotlin somehow, no clue what's happening here

Reply:
hrm. yeah, looks like jcmp doesn't know what to make of kotlin classes. :) I'd be ok with disabling it.